### PR TITLE
Added console-window

### PR DESCRIPTION
**Pull request by joeycatxx**

### CODE REVIEW

**Comment by prestonvanloon** [comment]
What does this do? 
Can you attach a screenshot?
Is there a bug associated with this task?

Thanks

**Comment by joeycatxx** [comment]
Added a new component called console-window.
Merged console-window and browser-window into one module - browser.module.

I think one of the folders src/app/browser did not get uploaded. That is why it is failing.

Using GitHub for the first time so I might have missed something.

File: src/app/exercise/playground/playground.component.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {SlideComponent} from '../../presentation/slide/slide.component';
   styleUrls: ['./playground.component.css']
 })
 export class PlaygroundComponent extends ExerciseBase {
+  @Input() isConsole: boolean;
```
Comment (prestonvanloon) [inline src/app/exercise/playground/playground.component.ts]: This is input as false always, does it need to be an `@Input`?

File: src/app/exercise/runner/runner.component.html [hunk 2]
```diff
@@ -1,3 +1,6 @@
-<slides-browser-window>
+<slides-browser-window *ngIf="!isConsole">
   <div #runner class="runner"></div>
 </slides-browser-window>
+<slides-console-window *ngIf="isConsole">
+  <div #runnerConsole class="runner"></div>
```
Comment (prestonvanloon) [inline src/app/exercise/runner/runner.component.html]: Is this ever shown? It looks like `isConsole` is assigned to false and never reassigned.

File: src/app/exercise/runner/runner.component.ts [hunk 3]
```diff
@@ -258,7 +260,9 @@ export class RunnerComponent implements AfterViewInit, OnChanges, OnDestroy {
   }
 
   ngOnChanges(changes: SimpleChanges): void {
-    this.runCode(changes.files.currentValue, this.runnerType);
+    if(this.runnerElement!=null || this.runnerConsoleElement!=null){
```
Comment (prestonvanloon) [inline src/app/exercise/runner/runner.component.ts]: Isn't this always true? One of those elements should always exist right?

File: src/app/exercise/runner/runner.component.ts [hunk 4]
```diff
@@ -336,6 +340,10 @@ export class RunnerComponent implements AfterViewInit, OnChanges, OnDestroy {
   }
 
   ngAfterViewInit() {
+    if(this.runnerElement==null && this.runnerConsoleElement!=null){
```
Comment (prestonvanloon) [inline src/app/exercise/runner/runner.component.ts]: `if (!this.runnerElement && this.runnerConsoleElement) {`

**Comment by joeycatxx** [comment]
Uploaded the missing folder to  the branch. For the typescript​ exercise am passing isconsole as true to get the console window instead of browser window

**Comment by prestonvanloon** [comment]
Please run 

- `npm run lint`
- `npm test`

And fix issues.

**Comment by joeycatxx** [comment]
Ran lint and fixed errors.
Also executed npm test. Test results here. Havent used this yet so not sure if everything I changed passed or not.
[KarmaTest-1.pdf](https://github.com/AngularNYC/angular-presentation/files/995054/KarmaTest-1.pdf)
[KarmaTest-2.pdf](https://github.com/AngularNYC/angular-presentation/files/995056/KarmaTest-2.pdf)
[KarmaTest-3.pdf](https://github.com/AngularNYC/angular-presentation/files/995055/KarmaTest-3.pdf)

**Comment by kirjs** [comment]
Please merge master: `git pull origin master`

**Pull request merged**